dla_cnn/desi/DesiMock.py

```diff
@@ -65,36 +70,32 @@ def read_fits_file(self, spec_path, truth_path, zbest_path, normalize = False):
         ivar_r = spec[9].data.copy()
         ivar_z = spec[14].data.copy()
         error = 1./np.sqrt(np.hstack((ivar_b,ivar_r,ivar_z)))
-        self.split_point_br = flux_b.shape[1]
-        self.split_point_rz = flux_b.shape[1]+flux_r.shape[1]
+        self.split_point_br = len(flux_b)
+        self.split_point_rz = len(flux_b)+len(flux_z)
         z_qso = zbest[1].data['Z'].copy()
         ra = spec[1].data['TARGET_RA'].copy()
         dec = spec[1].data['TARGET_DEC'].copy()
 
         self.data = {spec_id[i]:{'FLUX':flux[i],'ERROR': error[i], 'z_qso':z_qso[i] , 'RA': ra[i], 'DEC':dec[i], 'DLAS':spec_dlas[spec_id[i]]} for i in range(len(spec_id))}
-
-        if normalize:
-            err_spec = ''
-            for spec,data in self.data.items():
-                rest_wavelength = self.wavelength/(data['z_qso']+1)
-                if rest_wavelength[0]<1450:
-                    test = (rest_wavelength>=1400)&(rest_wavelength<=1500)
-                    data['FLUX'] = data['FLUX']/np.abs((np.median(data['FLUX'][test])))
-                else:
-                    err_spec = err_spec+"spectra'id:%i redshift:%f \n"%(spec,data['z_qso'])
-            assert err_spec == '', "Something wrong with spectrum below, please check them: \n"+err_spec
-            
+        
+        ndlas = []
```

Comment:
I see no reason to remove the spectra here.  
At the least, make it optional and default to not doing it.

We will choose which quasars (and DLAs) to include
when building our training, test sets, etc.

Reply:
Emmm, I remain those spectra, but some of them may not contain the Lymann forest part(3800/(1+z_qso)>1170, and in that case, the normalize method will assert error. So the users may need some test when using the normalize method. Maybe we can drop those spectrum with z<2.3, so all spectrum in DesiMock can be normalized.

Reply:
No, best that we error out if we try to normalize a spectrum that we shouldn't.